price_pusher/src/solana/solana.ts

```diff
@@ -78,13 +78,17 @@ export class SolanaPricePusher implements IPricePusher {
     const transactionBuilder = this.pythSolanaReceiver.newTransactionBuilder({
       closeUpdateAccounts: true,
     });
-    transactionBuilder.addUpdatePriceFeed(priceFeedUpdateData, this.shardId);
+    await transactionBuilder.addUpdatePriceFeed(
+      priceFeedUpdateData,
+      this.shardId
+    );
 
     try {
       await this.pythSolanaReceiver.provider.sendAll(
         await transactionBuilder.buildVersionedTransactions({
           computeUnitPriceMicroLamports: this.computeUnitPriceMicroLamports,
-        })
+        }),
+        { preflightCommitment: "processed" }
```

Comment:
Update preflightCommitment to avoid simulation errors

Reply:
why not skip it entirely :?

Reply:
Ok